model/host/host.go

```diff
@@ -894,6 +899,19 @@ func (h *Host) setStatusAndFields(ctx context.Context, newStatus string, query,
 		query,
 		update,
 	); err != nil {
+		// No documents matched: the database status has diverged from our cached status.
+		// Another job has already modified the host. Skip this update silently since our
+		// cached state is stale and attempting this status change is no longer valid.
+		if adb.ResultsNotFound(err) {
+			grip.Info(message.Fields{
+				"message":          "host status update skipped due to concurrent modification",
```

Comment:
I was wondering if this was necessary because we're probably not going to alert on this